include/net/ethernet.h

```diff
@@ -65,6 +65,7 @@ struct net_eth_addr {
 #define NET_ETH_PTYPE_PTP		0x88f7
 #define NET_ETH_PTYPE_LLDP		0x88cc
 #define NET_ETH_PTYPE_ALL               0x0003 /* from linux/if_ether.h */
+#define NET_ETH_PTYPE_ECAT		0x88a4
```

Comment:
Please call this `NET_ETH_PTYPE_ETHERCAT`

Reply:
SOME uses the pattern 'ECAT', I think it's brief and not ambiguous. it must be changed to 'ETHERCAT'?

Reply:
Hmm, I see both being used in different opensource implementations. Lets go with shorter one then as that is used in SOEM.